lms/djangoapps/onboarding_survey/models.py

```diff
@@ -2,7 +2,38 @@
 Models to support the on-boarding surveys
 """
 from django.db import models
+from django import forms
 from django.contrib.auth.models import User
+from student.models import UserProfile
+
+
+class Organization(models.Model):
+    """
+    Represents an organization.
+    """
+
+    name = models.CharField(max_length=255, db_index=True)
+    created = models.DateTimeField(auto_now_add=True, null=True, db_index=True)
+    point_of_contact_exist = models.BooleanField(default=False)
+
+    def __str__(self):
+        return self.name
+
+
+class ExtendedProfile(models.Model):
+    """
+    Model for extra fields in registration form.
+    """
+    POC_CHOICES = ((1, 'Yes'), (0, 'No'))
+
+    first_name = models.CharField(max_length=256)
+    last_name = models.CharField(max_length=256)
+    organization = models.ForeignKey(Organization, related_name='extended_profile', blank=True, null=True)
+    is_poc = models.BooleanField(choices=POC_CHOICES, default=False)
+    is_currently_employed = models.BooleanField(default=False)
+    user = models.OneToOneField(User, unique=True, db_index=True, related_name='extended_profile')
+    org_admin_email = models.EmailField(blank=True, default="")
+    are_surveys_complete = models.BooleanField(default=False)
```

Comment:
Can we rename this to `completed_survey`?

Reply:
Done